lib/media.js

```diff
@@ -351,6 +407,7 @@ function injectScriptsAndStyles(state) {
     return bluebird.props(mediaMap)
       .then(combinedFiles => {
         html = combinedFiles.styles ? appendMediaToTop(combinedFiles.styles, html) : html; // If there are styles, append them
+        html = combinedFiles.packs ? appendMediaToBottom(combinedFiles.packs, html) : html; // If there are packs, append them
```

Comment:
this allows assets to be inlined as directed by a manifest, and also allows the usage of the manifest to be optional, and therefore avoid breaking changes?

are assets currently inlined based on being defined in the compiled `/public` directory? or what are the circumstances in which we could accidentally **double** the number of inlined assets by using the manifest + existing compilation flow?